src/main/java/core/mate/academy/service/ExcavatorProduce.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProduce implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        excavators.add(new Excavator());
+        excavators.add(new Excavator());
```

Comment:
Here, you could also initialize new Excavator() with field values, like in line 11-12 above